tests/unit/ui/components/TestRating.js

```diff
@@ -253,22 +244,66 @@ describe(__filename, () => {
       });
       expect(allDivs).toBeTruthy();
     });
+
+    it('renders an appropriate title with an undefined rating when read-only', () => {
+      const root = render({ readOnly: true });
+
+      expect(findDOMNode(root).title).toEqual('This add-on has not been rated yet.');
+    });
+
+    it('renders an appropriate title with a null rating when read-only', () => {
+      const root = render({ rating: null, readOnly: true });
+
+      expect(findDOMNode(root).title).toEqual('This add-on has not been rated yet.');
+    });
+
+    it('renders an appropriate title with a given rating when read-only', () => {
+      const root = render({ rating: 3.8, readOnly: true });
+
+      expect(findDOMNode(root).title).toEqual('Rated 3.8 out of 5.');
+    });
+
+    it('renders an accessible description for null ratings and read-only', () => {
+      const root = render({ rating: null, readOnly: true });
+
+      expect(findDOMNode(root).title).toContain('This add-on has not been rated yet.');
+    });
+
+    it('renders read-only selected stars', () => {
+      const root = render({ rating: 3, readOnly: true });
+
+      // Make sure only the first 3 stars are selected.
+      [1, 2, 3].forEach((rating) => {
+        expect(root.ratingElements[rating].className).toEqual('Rating-choice Rating-selected-star');
+      });
+      [4, 5].forEach((rating) => {
+        expect(root.ratingElements[rating].className).toEqual('Rating-choice');
+      });
+    });
+
+    it('renders an accessible description for read-only ratings', () => {
+      const root = render({ rating: 3, readOnly: true });
+
+      [1, 2, 3, 4, 5].forEach((rating) => {
+        expect(root.ratingElements[rating].title).toEqual('Rated 3 out of 5.');
+      });
+    });
   });
 
   describe('rating counts', () => {
     const getRating = (props = {}) => findDOMNode(render(props)).textContent;
 
     it('renders the average rating', () => {
-      expect(getRating({ rating: 3.5 })).toEqual('Rated 3.5 out of 5');
+      expect(getRating({ rating: 3.5, readOnly: true })).toEqual('Rated 3.5 out of 5.');
     });
 
     it('localizes average rating', () => {
       const i18n = fakeI18n({ lang: 'de' });
-      expect(getRating({ rating: 3.5, i18n })).toContain('3,5');
+      expect(getRating({ rating: 3.5, i18n, readOnly: true })).toContain('3,5');
     });
 
     it('renders empty ratings', () => {
-      expect(getRating({ rating: null })).toEqual('Click to rate this add-on');
+      expect(getRating({ rating: null, readOnly: true })).toEqual('This add-on has not been rated yet.');
```

Comment:
You should try to wrap lines at max 80 chars.

Reply:
Thanks for that - I am wondering why linting isn't giving me an error.. locally or via Travis.

Reply:
I dont think we have a rule for that.

Reply:
It might be worthwhile to use [Prettier pre-commit hooks](https://prettier.io/docs/en/precommit.html) to automatically format code  when commits are made based on the existing `eslint` config.